app/src/features/surveys/view/SurveyAnimals.tsx

```diff
@@ -132,7 +200,7 @@ const SurveyAnimals: React.FC = () => {
             <Typography component="span" variant="subtitle1" color="textSecondary" mt={2}>
               {`${
                 animalCount
-                  ? `${animalCount} ${pluralize(animalCount, 'Animal')} reported in this survey`
```

Comment:
I think I wrote this line when I replaced the inline `pluralize` defn with the one found in `Utils`. This didn't work?